web/locales/es.json

```diff
@@ -104,28 +104,28 @@
         "legend": "Leyenda",
         "faq": "Preguntas frecuentes"
     },
-    "wind": "eólica",
-    "solar": "solar",
-    "hydro": "hidráulica",
+    "wind": "energía eólica",
+    "solar": "energía solar",
+    "hydro": "energía hidráulica",
```

Comment:
Not sure if we want the full title here, we also haven't done this for any other languages :)

Reply:
Yeah, I should have explained this :P

I added "energía" because it goes better with "X% of electricity is produced by...", which I took from the French translation because it was the formula that sounded best in Spanish compared to the Italian and Portuguese translations of the line. In French, though, they didn't add "energy" to the energy sources, and I don't know how it sounds in French, but in Spanish it sounds wrong to say ""X% of electricity is produced by solar". Your mind just finds something missing there.

That's why I added "energía", which I considered shortening to "en." but in the end decided against it (I counted the letters and saw it was no longer than battery storage ("almac. en baterías"), for example. But it could perfectly be shortened, if that's a problem :)

Reply:
Makes good sense for the tooltips, I'm mostly just wondering about the labels here:

![Screenshot 2021-03-08 at 13 17 14](https://user-images.githubusercontent.com/3296643/110320499-ade17c80-8010-11eb-9eff-83d170a1d3d7.png)

Wouldn't you have to also prefix nuclear, etc. with `energía` in that case?

What's your opinion on this, @Kongkille? :)

Reply:
Yup, seems like I missed nuclear. The rest would be good as is.

Reply:
Would it make sense to add 'energía' to the tooltip text instead?

Reply:
I would also prefer it without energia, but of course, I don't have the same context as @CeleminHero. I like @veqtrus idea of potentially putting it in the tooltip? wdyt?

Reply:
Hm... nope, I don't think it works if we put it in the tooltip.

Portuguese has it in the tooltip, but I would say it doesn't make too much sense in Portuguese either (energia goes well with solar, wind, hydro, geothermal and nuclear, which in Spanish and Portuguese are adjectives, but not with nouns such as coal or battery storage). That's why, to avoid having to say "geothermal energy", I chose a noun (geotermia) instead of the adjective (geotérmica).

I think the way I did it is the one that sounds best, but if there are problems I'd say either shorten energia to "en.", or go back to the previous translation, which was pretty understandable too.

Reply:
In Greek I also used adjectives but it's not a problem in the tooltip. The text is "**X** % of electrical energy available in **Y**  comes from **Z**". If **Z** is an adjective it goes with the noun 'energy', which is feminine in Greek. 'Source' is also feminine so it goes nicely together.

Reply:
Hmm yeah that's a clever solution. I'm still not too sure what the problem with "energía" is, but changing "electricidad" into "energía eléctrica" in the tooltip would certainly make it sound better.

Reply:
I think we are mainly trying to optimize so the labels aren't too large horizontally. If there's a way you think we can avoid having energia in label while still making it understandable, I think we should go for that.